internal/sim/components.go

```diff
@@ -22,24 +22,121 @@ import (
 
 //go:generate ../../cmd/weaver/weaver generate .
 
-// This file contains components used to test the simulator in sim_test.go.
+// This file contains components used to test the simulator in sim_test.go. The
```

Comment:
Can we add "test" to the name of the file, to make it clear it's only used in local tests?

Reply:
Unfortunately not. If we do, then `weaver generate` ignores it.